CONTRIBUTING.md

```diff
@@ -1,3 +1,57 @@
+# About this Document
+
+## Summary:
+
+This document provides details on how to contribute code to the software project.
+
+## Target Audience
+
+This document is geared towards developers interested in contributing code to this project. It is **not** intended to be read by end-users of the software, those interested in the software architecture, or operators who are deploying the software.
```

Comment:
I think the second sentence here comes across as a bit negative – what would you say to emphasising more that end-users and operators may find it interesting, but that it is not *directed* at them?

Reply:
I like it! I'll word something up and push in a bit